examples/data-sources/redfish_idrac_attributes/data-source.tf

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright (c) 2023 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://mozilla.org/MPL/2.0/
+
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+terraform {
+  required_providers {
+    redfish = {
+      version = "~> 1.0.0"
+      source  = "registry.terraform.io/dell/redfish"
+    }
+  }
+}
```

Comment:
This should block should be moved out from individual resource and datasource examples and put in the provider example.

Reply:
Updated